src/async/bimap/bimap.spec.js

```diff
@@ -3,27 +3,37 @@
 import bimap from './bimap';
 
 describe('async/bimap', () => {
-  test('should only call left fn if promise rejects', async (): Promise<any> => {
-    const leftFn: Function = jest.fn();
+  test('should only call leftFn if promise rejects', async (done: Function): Promise<any> => {
```

Comment:
If you're going to change from try/catch to .then/catch the can you remove the async keywords?

Reply:
Will do